viewers/nuxeo-image-viewer.html

```diff
@@ -189,12 +207,38 @@
           }
         }
 
+        _getFitIcon(fitToViewer) {
+          return fitToViewer ? 'nuxeo:fit-to-real-size' : 'nuxeo:fit-to-viewer';
+        }
+
+        _getOriginalZoomRatio(){
+          if(this._el){
+            const canvasData = this._el.initialCanvasData;
+            return canvasData.width / canvasData.naturalWidth;
+          }
+        }
+
         _resize() {
           if (this._el) {
             this._el.resize();
+            this._fitToViewer = true;
           }
         }
 
+        _verifyZoomRatio(data){
+          if(this._el){
+            // Cropper.js not always return the same number of decimal places and rounding.
+            // In order to ensure our calculations, we will use 5 decimal places.
+            const decimalPlaces = 5;
+
+            if(this._getOriginalZoomRatio().toFixed(decimalPlaces) === data.ratio.toFixed(decimalPlaces)){
```

Comment:
`this._fitToViewer = this._getOriginalZoomRatio().toFixed(decimalPlaces) === data.ratio.toFixed(decimalPlaces);`?